apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerFixtureBasePostgreSql.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2022 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using Google.Cloud.Spanner.Common.V1;
+using Google.Cloud.Spanner.V1.Internal.Logging;
+
+namespace Google.Cloud.Spanner.Data.CommonTesting
+{
+    /// <summary>
+    /// Base classes for test fixtures for Spanner PostgreSQL database.
+    /// </summary>
+    public abstract class SpannerFixtureBasePostgreSql : CloudProjectFixtureBase
+    {
+        public SpannerTestDatabasePostgreSql Database { get; }
+
+        public SpannerFixtureBasePostgreSql()
+        {
+            GrpcInfo.EnableSubchannelCounting();
```

Comment:
I would suggest removing this line, unless we really need subchannel counting in Postgres tests.
(This line would need to be conditionalized for GAX v4, as GrpcInfo only works with Grpc.Core.)